app/lib/rocdev_web/controllers/slack_controller.ex

```diff
@@ -0,0 +1,22 @@
+defmodule RocdevWeb.SlackController do
+  use RocdevWeb, :controller
+
+  alias Rocdev.ChatInviter
+
+  def index(conn, _params) do
+    redirect conn, to: page_path(conn, :index)
```

Comment:
Did you mean to redirect this to the register route?  If you don't want `/slack/` to  do anything you could just not create the route.

Reply:
Right now, the only place with the form is the home page so I'd like to go back there. I'm thinking of making a `/slack` to display activity stats and other fun things (same goes for the other social sites we have). That might also have the registration form. The idea is still a bit nebulous though.